src/pretix/control/templates/pretixcontrol/event/index.html

```diff
@@ -16,6 +16,42 @@ <h1>
             {% endif %}
         </small>
     </h1>
+    <div class="helper-space-below">
+        {% trans "Shop URL:" %}
+        <span id="shop_url" class="text-muted">{% abseventurl request.event "presale:event.index" %}</span>
+        <button type="button" class="btn btn-default btn-xs btn-clipboard" data-clipboard-target="#shop_url">
+            <i class="fa fa-clipboard" aria-hidden="true"></i>
+            <span class="sr-only">{% trans "Copy to clipboard" %}</span>
```

Comment:
Might be overkill, but probably better to understand.
```suggestion
            <span class="sr-only">{% trans "Copy shop URL to clipboard" %}</span>
```

Reply:
Do you think it is worth it? with my text, the same text is used in other places. I tend to try to avoid introducing new strings that can be avoided because translating pretix becomes more impossible every day

Reply:
Yeah, I do not think it is worth it. That’s why I added the „overkill“-part. Same thought, less strings to translate. But just wanted to make sure. So, leave it like that. :)